container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -232,6 +232,16 @@ function run_rpm_tempest {
 function generate_test_results {
     pushd $TEMPEST_DIR
 
+    echo "Excluded tests"
+    if [ ! -z ${TEMPEST_EXCLUDE_LIST} ]; then
+        echo "$(cat ${TEMPEST_EXCLUDE_LIST})"
```

Comment:
Is the `echo` here really necessary?:)

Reply:
Thanks for review, it is not needed. My concern with missing end of line was caused by a small mistake in documentation.